src/fake-timers-src.js

```diff
@@ -967,6 +968,11 @@ function withGlobal(_global) {
         // Prevent multiple executions which will completely remove these props
         clock.methods = [];
 
+        for (const [listener, signal] of clock.abortListenerMap.entries()) {
+            signal.removeEventListener("abort", listener);
+            clock.abortListenerMap.delete(listener);
```

Comment:
Modifying while iterating over maps appears to be [safe](https://stackoverflow.com/questions/35940216/es6-is-it-dangerous-to-delete-elements-from-set-map-during-set-map-iteration)

Reply:
It is